Project-I-Social-Card/social-card/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "moment": "^2.24.0",
```

Comment:
great job finding about moment, it is an amazing library that you will want to keep going back to.